### ISSUE TITLE
Enhance lint by shellcheck and hadolint

### ISSUE THREAD

**Issue by impactaky** [commaai/openpilot#32091]
What do you think about adding support for shellcheck to lint shell scripts and hadolint to lint Dockerfiles? 
If there are no negative perceptions about these tools, I'd be happy to work on creating a PR to implement this enhancement.

**Comment by adeebshihadeh** [comment]
Nope, no negative perceptions. If you want to do a PR, needs to be clean, simple, and give examples of the kinds of bugs it would catch.

Closing since this isn't a bug.
